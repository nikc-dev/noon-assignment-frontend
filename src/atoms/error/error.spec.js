import React from 'react';
import renderer from 'react-test-renderer';
import Error from './index';

it('renders correctly', () => {
  const component = renderer.create(
    <Error />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});