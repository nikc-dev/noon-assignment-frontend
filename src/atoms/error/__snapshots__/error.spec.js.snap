// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="error"
>
  <p>
    Knock! Knock! - Who's There? - No one  :(
  </p>
</div>
`;
